### ISSUE TITLE
Unable to compute metrics for model output/interp_cut-lemon1_batch_4_trans_0.8_reg_0.0001_layer_4_random

### ISSUE THREAD

**Issue by shenqianxiang** [PatrickDDj/TimeFormer-Code#1]
root@autodl-container-3cb043a06c-aa413778:~/TimeFormer-Code# python metrics.py -m output/interp_cut-lemon1_batch_4_trans_0.8_reg_0.0001_layer_4_random
Setting up [LPIPS] perceptual loss: trunk [vgg], v[0.1], spatial [off]
/root/miniconda3/lib/python3.10/site-packages/torchvision/models/_utils.py:208: UserWarning: The parameter 'pretrained' is deprecated since 0.13 and may be removed in the future, please use 'weights' instead.
  warnings.warn(
/root/miniconda3/lib/python3.10/site-packages/torchvision/models/_utils.py:223: UserWarning: Arguments other than a weight enum or `None` for 'weights' are deprecated since 0.13 and may be removed in the future. The current behavior is equivalent to passing `weights=VGG16_Weights.IMAGENET1K_V1`. You can also use `weights=VGG16_Weights.DEFAULT` to get the most up-to-date weights.
  warnings.warn(msg)
Downloading: "https://download.pytorch.org/models/vgg16-397923af.pth" to /root/.cache/torch/hub/checkpoints/vgg16-397923af.pth
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 528M/528M [00:03<00:00, 144MB/s]
Loading model from: /root/miniconda3/lib/python3.10/site-packages/lpips/weights/v0.1/vgg.pth

Scene: output/interp_cut-lemon1_batch_4_trans_0.8_reg_0.0001_layer_4_random
Method: ours_20000
Unable to compute metrics for model output/interp_cut-lemon1_batch_4_trans_0.8_reg_0.0001_layer_4_random

**Comment by shenqianxiang** [comment]
已解决
